src/flatc.cpp

```diff
@@ -1016,19 +930,6 @@ int FlatCompiler::Compile(const FlatCOptions &options) {
 
   std::unique_ptr<Parser> parser = GenerateCode(options, conform_parser);
 
-  // Once all the files have been parsed, run any generators Parsing Completed
-  // function for final generation.
-  for (size_t i = 0; i < params_.num_generators; ++i) {
```

Comment:
Ah, this will still be needed.

Check if there are any implementations of `parsing_completed` function and we might have to add a new method to the `CodeGenerator` interface to support those.

Reply:
None of the pre-refactored Generator functions have parsing_completed function set so I assume it's not used (https://github.com/google/flatbuffers/blob/flatbuffers-64/src/flatc_main.cpp#L81). Is it really used in the codebase?

Reply:
Done